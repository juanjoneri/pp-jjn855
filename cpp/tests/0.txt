H1 H2 H3
a b 10
1 -2 b
0.1 10.99 -99.9