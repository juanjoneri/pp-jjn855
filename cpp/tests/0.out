H1 H2 H3
a b 10
0.1 10.99 -99.9
